### PR TITLE
Improve tests

### PR DESCRIPTION
**Pull request by quodlibetor**
This PR is just a small thanks for providing the library, I don't need you to merge it on my behalf. Let me know if you'd like me to change anything.

This has two commits:

* Add SECRET_KEY to django config

  When I did `poetry run python runtests.py` Django complained that the
  SECRET_KEY was required. After I ran tests once it doesn't seem to matter, but
  this was a small speedbump that I thought it might be nice for future users to
  not encounter.
* Configure tox to test against Django 4.1

  Since it's out now and it works.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -13,10 +13,12 @@ jobs:
     strategy:
       matrix:
         python-version: [3.7, 3.8, 3.9]
-        django-version: [22, 32, 40]
+        django-version: [22, 32, 40, 41]
         exclude:
           - python-version: 3.7
             django-version: 40
+          - python-version: 3.7
```
Comment (pcraciunoiu) [inline .github/workflows/ci.yml:20]: Maybe we should move to python 3.10 as the default, and add it to the matrix too. I'm not sure if 3.7 support is still warranted, but I'm fine keeping it for now.

**Review by pcraciunoiu — COMMENTED** [review]
LGTM just a small note

File: .github/workflows/ci.yml (hunk 1 above)
Comment (quodlibetor) [reply .github/workflows/ci.yml:20]: I'm not sure how django-ses selects the default Python, but I'm happy to change it. I've added it to the gitlab test suite.

> I'm not sure if 3.7 support is still warranted, but I'm fine keeping it for now.

Yeah I wanted to be conservative and I didn't need any changes that would require bumping the version. I'm happy to remove 3.7 from tests and pypi classifiers if you'd like.

**Review by pcraciunoiu — APPROVED** [review]
Thank you for these improvements!

**Pull request merged**